benchmarks-log4j2/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's rename this file to `log4j2-benchmarks.xml`.   In case file with simlar name `log4j2.xml`  is already presebnt at classpath.

Reply:
WDYT?